### DISCUSSION TITLE
Workflow Archival

### DISCUSSION THREAD

**Discussion by muthurajr** [Netflix/conductor#2313] [General]
With the 3.0 release, I could see below settings are void for archiving workflow execution data to secondary storage.
conductor.additional.modules=com.netflix.conductor.contribs.ArchivingWorkflowModule
"workflowStatusListenerEnabled": true, 
https://github.com/Netflix/conductor/blob/v2.31.4/contribs/src/main/java/com/netflix/conductor/contribs/ArchivingWorkflowModule.java

Can you please help on few details on how to enable it and what other settings are required?
https://github.com/Netflix/conductor/blob/v3.0.6/contribs/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWorkflowListenerConfiguration.java
https://github.com/Netflix/conductor/blob/v3.0.6/contribs/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWorkflowStatusListener.java

**Comment by apanicker-nflx** [comment]
Please refer the [Changelog](https://github.com/Netflix/conductor/blob/main/CHANGELOG.md) for the relevant property changes. To enable this listener, please set `conductor.workflow-status-listener.type=archive` as per this [implementation](https://github.com/Netflix/conductor/blob/main/contribs/src/main/java/com/netflix/conductor/contribs/listener/archive/ArchivingWorkflowListenerConfiguration.java#L24).
